server/config/passport.js

```diff
@@ -9,21 +9,13 @@ module.exports = passport.use(
     try {
       const user = await User.findOne({ username });
       if (!user) {
-        return done(null, false, {
-          message: "Username does not exist",
-          name: "username",
-        });
+        return done(null, false);
```

Comment:
Does this also return the appropriate http status code?

Reply:
By default it returns 401 unless user is found. I could add messages into a custom callback rather than generating them in the client if you think that would be better practice?